sleap_nn/training/model_trainer.py

```
@@ -144,9 +147,10 @@ def __init__(
             try:
                 Path(self.dir_path).mkdir(parents=True, exist_ok=True)
             except OSError as e:
-                raise OSError(
+                logger.error(
                     f"Cannot create a new folder in {self.dir_path}. Check the permissions to the given Checkpoint directory. \n {e}"
                 )
+                raise OSError()
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)